src/data/formatters.js

```diff
@@ -16,6 +16,19 @@ export const formatDateRange = (dateRange: DateRange) =>
 
 export const formatTime = (value: string) => formatDate(value, "HH:mm");
 
+export const formatContentfulDate = (
+  year: string,
+  month: string,
+  day: string,
+  time?: string
+) => {
+  const correctedDay = day.length < 2 ? `0${day}` : day;
+  const correctedMonth = month.length < 2 ? `0${month}` : month;
+  const correctedYear = year.length < 4 ? `20${year}` : year;
```

Comment:
FYI, JavaScript has [`String.prototype.padStart`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) now, so this could be:
```js
day.padStart(2, "0")
month.padStart(2, "0")
year.padStart(4, "2000")
```

This is available in Node.js 9 and all browsers. Not sure what we can use in React Native, though.

Reply:
Oh cool. I'll try it out.